docs/_includes/intrinsic_attributes.adoc

```diff
@@ -0,0 +1,467 @@
+|<<document-title>>
+
+|nowrap
+|Use scrolling instead of wrapping in wide code listings.
+|(not set)
+|Any (value not used)
+|<<to-wrap-or-to-scroll>>
+
+|noxmlns
+|Remove the namespace from the Docbook 5 XML output.
+|(not set)
+|Any (value not used)
+|<<docbook>>
+
+|prewrap
+|Wrap wide code listing by default. (See also nowrap).
+|(empty)
+|Any (value not used)
+|<<to-wrap-or-to-scroll>>
+
+|scriptsdir
+|Sets the path to JavasScript files
+|(not set) in my tests, ./javascripts in text
+|Path
+|<<setting-attributes-on-a-document>>
+
+|sectanchors
+|Adds an anchor (empty link) before the section title.
+|(not set)
+|Any (value not used)
+|<<anchors>>
+
+|sectids
+|Synthesises IDs for any sections that do not already have one.
+|(empty)
+|Any (value not used)
+|<<auto-generated-ids>>
+
+|sectlinks
+|Turns section titles into links.
+|(not set)
+|Any (value not used)
+|<<links>>
+
+|showtitle
+|Toggles the display of an embedded document’s title
+|(not set)
+|Any (value not used)
+|<<document-title>>
+
+|source-highlighter
+|Source code highlighter to use.
+|(not set)
+|coderay, highlightjs, prettify, pygments
+|<<source-code-blocks>>
+
+|source-language
+|Set the default language for source code listings.
+|(not set)
+|Defined by the source highlighter in use, but includes at least java, c, ruby, xml.
+|<<source-code-blocks>>
+
+|split
+|Registers the split module for use in the document, Deck-JS backend only.
+|(not set)
+|Any (value not used)
+|<<deckjs-backend-attributes>>
+
+|status
+|Renders the current slide number and total number of slides
+|(not set)
+|Any (value not used)
+|<<deckjs-backend-attributes>>
+
+|stem
+|Set the processor used to render mathematics.
+|(not set)
+|asciimath, latexmath
+|<<stem-in>>
+
+|style
+|
+|(not set)
+|normal (default, so does not need to be set),
+literal,
+verse,
+quote,
+listing,
+TIP,
+NOTE,
+IMPORTANT,
+WARNING,
+CAUTION,
+abstract,
+partintro,
+comment,
+example,
+sidebar,
+source
+|<<style>>
+
+|stylesdir
+|Location for CSS style sheets.
+|.
+|Path
+|<<creating-a-theme>>
+
+|stylesheet
+|CSS stylesheet to replace the default one.
+|(empty)
+|File name
+|<<applying-a-theme>>
+
+|table-caption
+|Text of the label that is automatically prefixed to table titles. 
+To turn off figure caption labels and numbers, add the table-caption attribute to the document header and unset it.
+|Table
+|Any
+//Needs documenting, I guessed this from figure-caption
+|-
+
+|tip-caption
+|Sets the text used to label TIP admonitions, if icons are not used.
+|Tip
+|Any
+|<<admonition>>
+
+|toc
+|	
+|auto
+|auto, left, right, macro, preamble
+|<<user-toc>>	
+
+|toclevels
+|Maximum section level to display.
+|(not set = 2)
+|1,2,3,4,5
+|<<user-toc>>	
+
+|toc-title
+|Title for the TOC.
+|Table of Contents
+|Any
+|<<user-toc>>	
+
+|toc-placement
+|Deprecated: Use toc attribute instead.	
+|auto
+|
+|<<user-toc>>
+
+|toc-position
+|Deprecated: Use toc attribute instead.	
+|(not set = left)
+|left, right
+|<<user-toc>>
```

Comment:
Let's not mention `toc-placement` and `toc-position` as their use should be discouraged in favor of assigning a value to the `toc` attribute.